charts/pega/templates/pega-environment-config.yaml

```diff
@@ -59,6 +59,38 @@ data:
   CASSANDRA_TRUSTSTORE: "{{ .Values.dds.trustStore }}"
   # Path to a mounted key store file for Cassandra encryption
   CASSANDRA_KEYSTORE: "{{ .Values.dds.keyStore }}"
+  # Enable asynchronous processing of records in DDS Dataset save operation. Failures to store individual records will
+  # not interrupt Dataset save operations
+  CASSANDRA_ASYNC_PROCESSING_ENABLED: "{{ .Values.dds.asyncProcessingEnabled }}"
+  # Specify a prefix to use when creating Pega-managed keyspaces in Cassandra
+  CASSANDRA_KEYSPACES_PREFIX: "{{ .Values.dds.keyspacesPrefix }}"
+  # Enable an extended token aware policy for use when a Cassandra range query runs. When enabled this policy selects a
+  # token from the token range to determine which Cassandra node to send the request. Before you can enable this policy,
+  # you must configure the token range partitioner.
+  CASSANDRA_EXTENDED_TOKEN_AWARE_POLICY: "{{ .Values.dds.extendedTokenAwarePolicy }}"
+  # Enable a latency awareness policy, which collects the latencies of the queries for each Cassandra node and maintains
+  # a per-node latency score (an average).
+  CASSANDRA_LATENCY_AWARE_POLICY: "{{ .Values.dds.latencyAwarePolicy }}"
+  # Enable the use of a customized retry policy for your Pega Platform deployment. After enabling this policy in your
+  # deployment configuration, Cassandra queries that timeout will be retried. The number of retries may be configured
+  # using the dynamic system setting (DSS): dnode/cassandra_custom_retry_policy/retryCount. If not configured, queries
+  # will be retried once
+  CASSANDRA_CUSTOM_RETRY_POLICY: "{{ .Values.dds.customRetryPolicy }}"
+  # Enable the speculative execution policy for retrieving data from your Cassandra service. When enabled, the Pega
+  # Platform will send a query to multiple nodes in your Cassandra service and process the first query response. This
+  # provides lower perceived latencies at the client, but puts greater load on your Cassandra service.
+  CASSANDRA_SPECULATIVE_EXECUTION_POLICY: "{{ .Values.dds.speculativeExecutionPolicy }}"
+  # Enable reporting of DDS SDK metrics to a Java Management Extension (JMX) format for use by your organization to
+  # monitor your Cassandra service. Setting this property `false` disables metrics being exposed through the JMX
+  # interface; disabling also limits the metrics being collected using the DDS landing page
```

Comment:
consistency...missing period>>interface; disabling also limits the metrics being collected using the DDS landing page.

Reply:
Fixed